doc/source/conf.py

```diff
@@ -21,6 +22,13 @@
 import recommonmark
 
 
+# -- Versioning with setuptools_scm ------------------------------------------
+from pkg_resources import get_distribution
+release = get_distribution('icepyx').version
+# for example take major/minor
+version = '.'.join(release.split('.')[:2])
+
+
```

Comment:
So do we need this still? Or can we get rid of it?

Reply:
Good question, I tried deleting these lines locally and running `cd icepyx/doc && make html` works. Could you try and remove them and see if the readthedocs build passes still?